src/VisualStudio/Core/Next/Remote/ServiceHubRemoteHostClient.cs

```diff
@@ -303,5 +329,36 @@ private static bool ReportNonIOException(Exception ex)
             // catch all exception. not worth crashing VS.
             return true;
         }
+
+        private static readonly TimeSpan s_reportTimeout = TimeSpan.FromMinutes(10);
+        private static bool s_timeoutReported = false;
```

Comment:
Maybe it is reasonable to move class vars to the top of the class

Reply:
I have put these together since it is diagnostic related code.